src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isSelected = tab.id === selectedTab.id;
+
+            return (
+              <li
+                className={classNames({ 'is-active': isSelected })}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    return isSelected
+                      ? null
+                      : onTabSelected(tab);
+                  }}
```

Comment:
we don't need to return null if the tab is not currently selected.
Try doing `isSelected && (...` - this should work. Or just usual `if`